src/app/app.component.html

```diff
@@ -9,15 +9,16 @@ <h2>HQ Dashboard <button class="btn btn-primary pull-right btn-print"><i class="
             </p>
             <div style="padding-top:30px; font-size: 1.1em;">
                 <ul class="list-inline">
-                    <li class="list-inline-item"><a class="active" href="#">LAST 7 DAYS</a></li>
-                    <li class="list-inline-item"><a href="#">7 DAYS FORECAST</a></li>
+                    <li class="list-inline-item"><a class="active" routerLink="/last-7-day-post-list" routerLinkActive="active">LAST 7 DAYS</a></li>
```

Comment:
I think we should use this syntax for router links: [routerLink]="['/home']", because some routes will have parameters passed to them which requires the brackets, and it would be good to stay consistent